src/main/java/frc/robot/subsystems/Scoring.java

```diff
@@ -20,7 +20,7 @@ public class Scoring extends PIDSubsystem {
     private final CANSparkMax rightScoringElevatorMotor = new CANSparkMax(
             ScoringConstants.rightScoringElevatorMotorPort, MotorType.kBrushless);
     private final CANSparkMax rollerMotor = new CANSparkMax(
-            ScoringConstants.rollerMotorPort, MotorType.kBrushless);
+            ScoringConstants.rollerMotorPort, MotorType.kBrushed);
```

Comment:
We should not commit any brushed motors. The chances of someone forgetting these and deploying to the comp bot and burning out neos is too high. Please set all instances of kBrushed back to Brushless before merging